# hil-data,0.40	status=REVIEW	provisional-reference difference; manual adjudication required
--- expected/hil-data,0.40
+++ actual/hil-data,0.40
@@ -1 +1 @@
-( ( IP-MAT ( NP ( NP ( D NaGani ) ( N wetiGa ) ) ( PUNC , ) ( CP-REL ( WNP-1 ( WPRO ane ) ) ( IP-SUB ( NP-TRACE ( -NONE- *T*-1 ) ) ( VB iwaGadi ) ) ) ( PUNC , ) ) ( VBAPL eniteloco ) ( NP-APL ( N$ iGonagi ) ) ) ( ID hil-data,0.40 ) )
+( ( IP-MAT ( NP ( D NaGani ) ( N wetiGa ) ) ( PUNC , ) ( NP ( CP-FRL ( WNP-1 ( WPRO ane ) ) ( IP-SUB ( NP-TRACE *T*-1 ) ( VB iwaGadi ) ) ) ) ( PUNC , ) ( VBAPL eniteloco ) ( NP-APL ( N$ iGonagi ) ) ) ( ID hil-data,0.40 ) )
# hil-data,0.8	status=DONE	validated-reference difference
--- expected/hil-data,0.8
+++ actual/hil-data,0.8
@@ -1 +1 @@
-( ( IP-MAT ( NP-SBJ ( D NiGijo ) ( N$ liwenigi ) ) ( NP-PRD ( N$ libinienigi ) ) ) ( ID hil-data,0.8 ) )
+( ( IP-MAT ( NP ( D NiGijo ) ( N$ liwenigi ) ( NP ( N$ libinienigi ) ) ) ) ( ID hil-data,0.8 ) )
# ped-gramm,0.12	status=DONE	validated-reference difference
--- expected/ped-gramm,0.12
+++ actual/ped-gramm,0.12
@@ -1 +1 @@
-( ( IP-MAT ( NP-SBJ ( D ica ) ( N$ looligi ) ) ( NP-PRD ( N$ lidi ) ) ) ( ID ped-gramm,0.12 ) )
+( ( IP-MAT ( NP ( D ica ) ( N$ looligi ) ( NP ( N$ lidi ) ) ) ) ( ID ped-gramm,0.12 ) )
# ped-gramm,0.15	status=DONE	validated-reference difference
--- expected/ped-gramm,0.15
+++ actual/ped-gramm,0.15
@@ -1 +1 @@
-( ( IP-MAT ( NP-SBJ ( D ica ) ( N$ looligi ) ) ( NP-PRD ( N$ lideGegi ) ) ) ( ID ped-gramm,0.15 ) )
+( ( IP-MAT ( NP ( D ica ) ( N$ looligi ) ( NP ( N$ lideGegi ) ) ) ) ( ID ped-gramm,0.15 ) )
# ped-gramm,0.16	status=DONE	validated-reference difference
--- expected/ped-gramm,0.16
+++ actual/ped-gramm,0.16
@@ -1 +1 @@
-( ( IP-MAT ( NP-SBJ ( N$ loigipodi ) ) ( NP-PRD ( N$ libinienigipi ) ) ) ( ID ped-gramm,0.16 ) )
+( ( IP-MAT ( NP ( N$ loigipodi ) ( NP ( N$ libinienigipi ) ) ) ) ( ID ped-gramm,0.16 ) )
# ped-gramm,0.17	status=REVIEW	provisional-reference difference; manual adjudication required
--- expected/ped-gramm,0.17
+++ actual/ped-gramm,0.17
@@ -1 +1 @@
-( ( IP-MAT ( NP ( D ica ) ( N$ loigi ) ( NP ( D ijo ) ( N niganigi ) ) ) ( VBAPL ipegitegi ) ( NP-APL ( N$ GanigotGa ) ) ) ( ID ped-gramm,0.17 ) )
+( ( IP-MAT ( NP ( NP ( D ica ) ( N$ loigi ) ) ( D ijo ) ( N niganigi ) ) ( VBAPL ipegitegi ) ( NP-APL ( N$ GanigotGa ) ) ) ( ID ped-gramm,0.17 ) )
# ped-gramm,0.19	status=DONE	validated-reference difference
--- expected/ped-gramm,0.19
+++ actual/ped-gramm,0.19
@@ -1 +1 @@
-( ( IP-MAT ( NP-SBJ ( D ica ) ( N$ liwigo ) ) ( NP-PRD ( N$ libinienigi ) ) ) ( ID ped-gramm,0.19 ) )
+( ( IP-MAT ( NP ( D ica ) ( N$ liwigo ) ( NP ( N$ libinienigi ) ) ) ) ( ID ped-gramm,0.19 ) )
# ped-gramm,0.29	status=REVIEW	provisional-reference difference; manual adjudication required
--- expected/ped-gramm,0.29
+++ actual/ped-gramm,0.29
@@ -1 +1 @@
-( ( IP-MAT ( NP ( N$ Eyodi ) ( CP-me ( C me ) ( IP-SUB ( NP ( N niganaGacanajo ) ) ) ) ) ) ( ID ped-gramm,0.29 ) )
+( ( IP-MAT ( NP-SBJ ( N$ Eyodi ) ) ( CP-me ( C me ) ( IP-SUB ( NP ( N niganaGacanajo ) ) ) ) ) ( ID ped-gramm,0.29 ) )
# ped-gramm,0.40	status=DONE	validated-reference difference
--- expected/ped-gramm,0.40
+++ actual/ped-gramm,0.40
@@ -1 +1 @@
-( ( IP-MAT ( NP-SBJ ( N$ iGeladi ) ) ( NP-PRD ( D digoida ) ( N weiigi ) ) ( PUNC . ) ) ( ID ped-gramm,0.40 ) )
+( ( IP-MAT ( NP ( NP ( N$ iGeladi ) ) ( D digoida ) ( N weiigi ) ) ( PUNC . ) ) ( ID ped-gramm,0.40 ) )
# ped-gramm,0.45	status=REVIEW	provisional-reference difference; manual adjudication required
--- expected/ped-gramm,0.45
+++ actual/ped-gramm,0.45
@@ -1 +1 @@
-( ( IP-MAT ( NP-SBJ ( NP ( NPR João ) ) ( N$ liGeladi ) ) ( NP-PRD ( N napioi ) ) ( PUNC . ) ) ( ID ped-gramm,0.45 ) )
+( ( IP-MAT ( NP-SBJ ( NPR João ) ) ( NP-PRD ( N$ liGeladi ) ( NP ( N napioi ) ) ) ( PUNC . ) ) ( ID ped-gramm,0.45 ) )
# ped-gramm,0.47	status=REVIEW	provisional-reference difference; manual adjudication required
--- expected/ped-gramm,0.47
+++ actual/ped-gramm,0.47
@@ -1 +1 @@
-( ( IP-MAT ( NP ( N$ liGeladi ) ) ( VBAPL ipegitege ) ( NP-APL ( NP ( N$ niweiigi ) ) ( N$ nigotaGa ) ) ( PUNC . ) ) ( ID ped-gramm,0.47 ) )
+( ( IP-MAT ( NP ( N$ liGeladi ) ) ( VBAPL ipegitege ) ( NP-APL ( N$ niweiigi ) ( NP ( N$ nigotaGa ) ) ) ( PUNC . ) ) ( ID ped-gramm,0.47 ) )
# van-data,0.1	status=DONE	validated-reference difference
--- expected/van-data,0.1
+++ actual/van-data,0.1
@@ -1 +1 @@
-( ( IP-MAT ( NP-SBJ ( N$ iGeladi ) ) ( NP-PRD ( D digoida ) ( N weiigi ) ) ) ( ID van-data,0.1 ) )
+( ( IP-MAT ( NP ( NP ( N$ iGeladi ) ) ( D digoida ) ( N weiigi ) ) ) ( ID van-data,0.1 ) )
# van-data,0.11	status=DONE	validated-reference difference
--- expected/van-data,0.11
+++ actual/van-data,0.11
@@ -1 +1 @@
-( ( IP-MAT ( NP-SBJ ( D niGida ) ( N$ liwenigi ) ) ( NP-PRD ( N$ libinienigi ) ) ) ( ID van-data,0.11 ) )
+( ( IP-MAT ( NP ( D niGida ) ( N$ liwenigi ) ( NP ( N$ libinienigi ) ) ) ) ( ID van-data,0.11 ) )
# van-data,0.30	status=DONE	validated-reference difference
--- expected/van-data,0.30
+++ actual/van-data,0.30
@@ -1 +1 @@
-( ( IP-MAT ( NP-SBJ ( D niGijo ) ( N$ lodaajo ) ( NP ( D ajo ) ( N$ lodowa ) ) ) ( NEG aG@ ) ( NP-PRD ( Q @ica ) ( D digoida ) ( N$ liGeladi ) ) ) ( ID van-data,0.30 ) )
+( ( IP-MAT ( NP-SBJ ( NP ( D niGijo ) ( N$ lodaajo ) ) ( D ajo ) ( N$ lodowa ) ) ( NEG aG@ ) ( NP-PRD ( Q @ica ) ( D digoida ) ( N$ liGeladi ) ) ) ( ID van-data,0.30 ) )
# van-data,0.37	status=DONE	validated-reference difference
--- expected/van-data,0.37
+++ actual/van-data,0.37
@@ -1 +1 @@
-( ( IP-MAT ( NP ( D ani ) ( N wetiGa ) ( CP-me ( C me ) ( IP-SUB ( VB iwaGadi ) ) ) ) ( VBAPL eniteloco ) ( NP-APL ( N$ iGonagi ) ) ) ( ID van-data,0.37 ) )
+( ( IP-MAT ( NP ( D ani ) ( N wetiGa ) ) ( CP-me ( C me ) ( IP-SUB ( VB iwaGadi ) ) ) ( VBAPL eniteloco ) ( NP-APL ( N$ iGonagi ) ) ) ( ID van-data,0.37 ) )
# van-data,0.44	status=DONE	validated-reference difference
--- expected/van-data,0.44
+++ actual/van-data,0.44
@@ -1 +1 @@
-( ( IP-MAT ( NP-SBJ ( D ijoa ) ( N nigetedi ) ( NP ( N niwigo ) ) ) ( NP-PRD ( N$ libinienigipi ) ) ) ( ID van-data,0.44 ) )
+( ( IP-MAT ( NP-SBJ ( D ijoa ) ( N nigetedi ) ) ( NP-PRD ( N niwigo ) ( N$ libinienigipi ) ) ) ( ID van-data,0.44 ) )
# van-data,0.73	status=REVIEW	provisional-reference difference; manual adjudication required
--- expected/van-data,0.73
+++ actual/van-data,0.73
@@ -1 +1 @@
-( ( IP-MAT ( NP ( N$ ligeladi ) ) ( VBAPL ipegitege ) ( NP-APL ( NP ( N$ niweiigi ) ) ( N$ nigotaGa ) ) ) ( ID van-data,0.73 ) )
+( ( IP-MAT ( NP ( N$ ligeladi ) ) ( VBAPL ipegitege ) ( NP-APL ( N$ niweiigi ) ( NP ( N$ nigotaGa ) ) ) ) ( ID van-data,0.73 ) )
